app/src/main/java/com/konkuk/codion/ui/myCloset/component/OutlineButton.kt

```diff
@@ -0,0 +1,44 @@
+package com.konkuk.codion.ui.myCloset.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.konkuk.codion.ui.theme.CodiOnTypography
+import com.konkuk.codion.ui.theme.Gray100
+import com.konkuk.codion.ui.theme.Gray700
+
+@Composable
+fun OutlineButton(
+    modifier: Modifier = Modifier,
+    label: String,
```

Comment:
냥니가 저번에 만들어둔 버튼 컴포넌트들에서는 text라는 변수명을 쓰고 있어서 얘두 통일시키는 게 좋을 듯?? 뭔가 label은 버튼 위에 추가로 달려있는 설명 같은.. 느낌..ㅎ..
```suggestion
    text: String,
```